src/main/pages/che-7/overview/assembly_che-quick-starts.adoc

```diff
@@ -20,22 +20,55 @@ summary:
 
 This section contains instructions for getting quickly started with {prod}.
 
-Various installation methods are covered:
+{prod-short} is available in two modes:
 
-* link:{site-baseurl}che-7/installing-the-{prod-cli}-management-tool/[Installing the {prod-cli} management tool]
+* *Single-user*: Non-authenticated {prod-short}, lighter and suited for personal environments
 
-* link:{site-baseurl}che-7/running-che-locally/[Running {prod-short} locally] on a personal workstation
+* *Multi-user*: Authenticated {prod-short} suited for organizations and developer teams
 
-* link:{site-baseurl}che-7/deploying-che-on-kubernetes-on-aws/[Deploying {prod-short} on Kubernetes on Amazon Elastic Compute Cloud]
+You need a Kubernetes (version 1.9 or higher) or OpenShift cluster to deploy {prod-short} on.
 
-* link:{site-baseurl}che-7/installing-che-on-openshift-3-using-the-operator/[Installing {prod-short} on OpenShift 3 using the Operator]
+Various installation methods are covered in this documentation.
+It goes from 
 
-* link:{site-baseurl}che-7/installing-che-on-openshift-4-from-operatorhub/[Installing {prod-short} on OpenShift 4 from OperatorHub]
+== On a local single-node cluster
 
-* link:{site-baseurl}che-7/installing-che-on-google-cloud-platform/[Installing {prod-short} on Google Cloud Platform]
+WARNING: Remember that single-node Kubernetes clusters are suited only for testing or single-user development.
+Do *NOT* use such clusters to run {prod-short} for organizations or developer teams.
 
-* link:{site-baseurl}che-7/installing-eclipse-che-on-microsoft-azure/[Installing {prod-short} on Microsoft Azure]
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-codeready-containers/[CodeReady containers]
 
-* link:{site-baseurl}che-7/accessing-che-from-openshift-developer-perspective/[Accessing {prod-short} from OpenShift Developer Perspective]
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-docker-desktop/[Docker Desktop]
+
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-minikube/[Minikube]
+
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-minishift/[Minishift]
+
+
+== On a private (__on-premise__) cluster
+
+Create your Kubernetes `production-capable` cluster _on-premise__ to run {prod-short}.
+
+NOTE: `production-capable` means you can deploy clusters from __test-like__ xref:on-a-local-single-node-cluster[local single-node clusters] to production grade high-availability ones depending on your configuration (number of nodes, etc.)
+
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-bare-metal-using-kubespray/[Bare metal] __using Kubespray__
+
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-openshift-3-using-the-operator/[OpenShift 3] __using the Operator__
+
+* link:{site-baseurl}che-7/installing-{prod-id-short}-on-openshift-4-using-operatorhub/[OpenShift 4] __using OperatorHub__
+
+== On a cloud provided cluster
```

Comment:
For the same reason mentioned above I would change here to "On a public cloud provider"

Reply:
fixed, thanks @l0rd